Source/AssistantV1/Assistant.swift

```diff
@@ -217,10 +222,11 @@ public class Assistant {
      new workspace.    This operation is limited to 30 requests per 30 minutes. For more information, see **Rate
      limiting**.
 
-     - parameter properties: The content of the new workspace.    The maximum size for this data is 50MB. If you need to import a larger workspace, consider importing the workspace without intents and entities and then adding them separately.
+     - parameter properties: The content of the new workspace.    The maximum size for this data is 50MB. If you need to import a larger
```

Comment:
I think these extra spaces will be fixed by the changes I have in the apiref branch.